opcua_server.c

```diff
@@ -98,6 +135,48 @@ static void add_tempsensors(void)
     }
 }
 
+static void add_ports(void)
+{
+    uint32_t countAll = 0;
```

Comment:
```suggestion
    uint32_t count_all = 0;
```

The other non-open62541 variables here use this naming scheme. Right/good or not, but consistency is king.